src/chessground/fen.ts

```diff
@@ -27,6 +27,36 @@ const letters = {
   king: 'k'
 }
 
```

Comment:
Cen we use read function to avoid almost the same code

Reply:
Maybe by using functors and template read functions. Lets leave it for refactor in future

Reply:
ok